src/components/ForgotUserName/ForgotUserNameCtrl.js

```diff
@@ -32,32 +41,33 @@ class ForgotUserNameCtrl extends Component {
   });
 
   state = {
-    isValidEmail: false,
+    isValidEmail: true,
     userExists: false,
-    hasErrorsContainer: false,
     userEmail: '',
   };
 
-  handleSubmit = values => {
-    this.resetState();
-    const {
-      mutator: {
-        searchUsername,
-      },
-    } = this.props;
-    const { userInput } = values;
-    const isValidInput = validateForgotUsernameForm(userInput);
+    handleSubmit = values => {
+      this.resetState();
+      const {
+        mutator: {
+          searchUsername,
+        },
+        stripes: {
+          store,
+        },
+      } = this.props;
+      const { userInput } = values;
+      const isValidInput = validateForgotUsernameForm(userInput);
 
-    return isValidInput
-      ? searchUsername
-        .POST({ id: userInput })
-        .then(() => this.handleSuccessfulResponse(userInput))
-        .catch(this.handleBadResponse)
-      : this.setState({
-        isValidEmail: false,
-        hasErrorsContainer: true,
-      });
-  };
+      return isValidInput
+        ? searchUsername
+          .POST({ id: userInput })
+          .then(() => this.handleSuccessfulResponse(userInput))
+          .catch(response => { processBadResponse(store, response); })
```

Comment:
Please consider rewriting without curly braces.
```suggestion
          .catch(response => processBadResponse(store, response))
```